sky/skylet/log_lib.py

```diff
@@ -425,15 +426,19 @@ def tail_logs(job_owner: str,
         with open(log_path, 'r', newline='') as log_file:
             # Using `_follow` instead of `tail -f` to streaming the whole
             # log and creating a new process for tail.
-            for line in _follow_job_logs(
-                    log_file,
-                    job_id=job_id,
-                    start_streaming_at='INFO: Tip: use Ctrl-C to exit log'):
+            for line in _follow_job_logs(log_file,
+                                         job_id=job_id,
+                                         start_streaming_at=start_stream_at):
                 print(line, end='', flush=True)
     else:
         try:
+            start_stream = False
             with open(log_path, 'r') as f:
-                print(f.read())
+                for line in f.readlines():
+                    if start_stream_at in line:
+                        start_stream = True
+                    if start_stream:
+                        print(line, end='', flush=True)
```

Comment:
Thanks for adding this! Is the purpose of this change to hide the output that comes before `INFO: Tip: use Ctrl-C`? I can still see it:

```
(base) romilb@romilbx1yoga:/mnt/d/Romil/Berkeley/Research/sky-experiments/examples$ sky cancel myclus 2
Cancelling jobs (2) on cluster 'myclus'...
(base) romilb@romilbx1yoga:/mnt/d/Romil/Berkeley/Research/sky-experiments/examples$ sky logs myclus 2
Tailing logs of job 2 on cluster 'myclus'...
I 10-30 02:32:35 log_lib.py:385] Start streaming logs for job 2.
2022-10-30 02:19:35,045 INFO worker.py:1227 -- Using address 172.31.35.172:6379 set in the environment variable RAY_ADDRESS
2022-10-30 02:19:35,045 INFO worker.py:1337 -- Connecting to existing Ray cluster at address: 172.31.35.172:6379...
2022-10-30 02:19:35,050 INFO worker.py:1513 -- Connected to Ray cluster. View the dashboard at 127.0.0.1:8265 
INFO: Tip: use Ctrl-C to exit log streaming (task will not be killed).
INFO: Waiting for task resources on 1 node. This will block if the cluster is full.
INFO: All task resources reserved.
INFO: Reserved IPs: ['172.31.35.172']
(task pid=26142) hi
```
Is this expected?

Reply:
Hmmm, that is not expected. Could you check the skypilot on your cluster to see if it has the latest update? If the cluster is launched before this PR, it may require a `sky launch` to update the skypilot on that cluster.

Reply:
Ahh yes this was a cluster launched from master branch. Re-launching from this branch fixed it. Thanks!